app/src/main/java/com/bogdan/codeforceswatcher/NotificationReceiver.kt

```diff
@@ -0,0 +1,56 @@
+package com.bogdan.codeforceswatcher
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.support.v4.app.NotificationCompat
+import com.bogdan.codeforceswatcher.activity.MainActivity
+
+class NotificationReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        onRefresh()
+        if (LoadUser.textNotification != "") {
```

Comment:
`onRefresh` could easily returned the `textNotification` so you didn't need to have a class variable for that.